web/web-mock/web-mock-rest/src/main/java/com/castlemock/web/mock/rest/controller/mock/AbstractRestServiceController.java

```diff
@@ -26,24 +25,14 @@
 import com.castlemock.model.core.utility.parser.ExternalInputBuilder;
 import com.castlemock.model.core.utility.parser.TextParser;
 import com.castlemock.model.core.utility.parser.expression.argument.ExpressionArgument;
-import com.castlemock.model.mock.rest.domain.RestEvent;
```

Comment:
I personally prefer to specify what is being imported instead of using *. It is more clear what is being imported.